### DISCUSSION TITLE
Collect information about Hardware, RTK and radio here?

### DISCUSSION THREAD

**Discussion by chrta** [mikey0000/PyMammotion#30] [General]
I would like to collect/document more information about my Luba 2 and how it works. Especially the main controller board, it's controller and connectors, the RTK station and the LoRa radio protocol.

This project already contains a lot information about the Luba. Should I add this here or create a new project for it?

**Comment by mikey0000** [comment]
Sounds like the wiki could be suitable. If you have documents on schematics or hacking the hardware happy for that to go in the repo, join us in the discord and can keep talking about it
